app/controllers/schools/find_out_more_controller.rb

```diff
@@ -21,5 +24,15 @@ def show
     def content_managed?
       @find_out_more.alert.alert_type.class_name == "Alerts::System::ContentManaged"
     end
+
+    def redirect_if_disabled_and_not_admin
+      return unless EnergySparks::FeatureFlags.active?(:replace_find_out_mores)
```

Comment:
maybe this could go with the `before_action`? e.g.
```
    before_action :redirect_if_disabled_and_not_admin, if: -> { EnergySparks::FeatureFlags.active?(:replace_find_out_mores) }
```

Reply:
Interesting, hadn't thought of that. I might leave is is for now as the idea is we'll remove all this code once we've launched the new pages.